apps/worker/src/app/workflow/workflow.module.ts

```diff
@@ -108,22 +108,10 @@ const USE_CASES = [
 const REPOSITORIES = [JobRepository];
 
 const SERVICES: Provider[] = [
-  {
-    provide: MetricQueueService,
-    useClass: MetricQueueService,
-  },
-  {
-    provide: QueueService,
-    useClass: WorkflowQueueService,
-  },
-  {
-    provide: TriggerQueueService,
-    useClass: TriggerProcessorQueueService,
-  },
-  {
-    provide: WsQueueService,
-    useClass: WsQueueService,
-  },
+  QueueService,
+  TriggerQueueService,
```

Comment:
It looks wrong to me (but I'm not sure) that we do provide: `QueueService, TriggerQueueService` and then also `WorkflowWorkerService, TriggerWorkerService`. This way we will have two instances of the same queues because the `WorkflowWorkerService extends QueueService` and `TriggerWorkerService extends TriggerQueueService`. The `WorkflowWorkerService, TriggerWorkerService` instances were injected by their base class because some use cases leave in the `application-generic` package but these classes are defined in the worker app.
I feel it's better to have what was there before.

Reply:
For future readers, I have been pairing with @LetItRock and we agreed to tackle the proposed refactor in a different PR, as the separation of concerns is a good thing and we can have it less confusing as right now. Same for these dependencies and renaming. We will clarify it in that other PR.